deployments/fuji/migrations/1644432723_deploy_fuji.ts

```diff
@@ -82,6 +84,9 @@ migration('1644432723_deploy_fuji', {
       contracts
     );
 
+    // Verify contracts after all contracts have been deployed
+    await deploymentManager.verifyContracts();
```

Comment:
Awesome feature, but this interface feels a little awkward...you have to set the option then you call verify at the end too, which doesn't seem to check the option?

Reply:
Yeah I can spend more time thinking about the interface. It's a bit clunky because the deployment manager doesn't know when it should verify contracts, so we always have to manually call it at some point.

Reply:
Added a `doThenVerify` function to `DeploymentManager` that should improve the interface a bit.